qa/t/webservice_user_create.t

```diff
@@ -13,13 +13,17 @@ use strict;
 use warnings;
 use lib qw(lib ../../lib ../../local/lib/perl5);
 use QA::Util;
-use Test::More tests => 87;
+use Test::More tests => 99;
 my ($config, $xmlrpc, $jsonrpc, $jsonrpc_get) = get_rpc_clients();
 
 use constant NEW_PASSWORD => 'UiX1Shuuchid';
 use constant NEW_FULLNAME => 'WebService Created User';
 
-use constant PASSWORD_TOO_SHORT => 'a';
+use constant PASSWORD_TOO_SHORT     => 'a';
+use constant PASSWORD_TOO_FEW_WORDS => 'bip bop boop';
+use constant PASSWORD_NOT_COMPLEX   => 'abcdefghijk1';
```

Comment:
This is missing a test for a valid pass phrase.

There should also be tests that exercise all of the possible valid combinations; eg. there isn't anything here that tests for symbols.

Perhaps this would be easier as a unit test?